src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,109 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int MULTIPLIER_CAPACITY = 2;
+    private int size;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size == threshold()) {
+            resize();
+        }
+        Node<K, V> identicalNode = findNode(key);
+        if (identicalNode != null) {
+            identicalNode.value = value;
+        } else {
+            insertNewValue(key, value);
+        }
     }
 
     @Override
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the `getValue` method.